components/Header.js

```diff
@@ -0,0 +1,22 @@
+import styled, { css } from "styled-components";
+
+const Head = styled.p`
```

Comment:
```suggestion
const Header = styled.header`
```

you should use a different name, there is a Head component provided by Next.js and this might lead to confusion.
(We did not talk about this component, so you cannot know this)

And you should use a header element instead of a p element.

Reply:
ok, i changed the naming up a bit, should be all clear now.